### DISCUSSION TITLE
kill a `mirai` with `saisei` function

### DISCUSSION THREAD

**Discussion by be-marc** [r-lib/mirai#171] [Q&A] [answered]
I would like to kill a `mirai` with the `saisei(force = TRUE)` function. I run n `marai`s on n `daemons`. I only want to kill one specific `mirai`. How do I know on which  `daemon` the `mirai` runs so I can set the `i` parameter of `saisei()`?

**Comment by shikokuchuo** [comment, marked as answer]
Hi Marc, that's a great idea, but we currently don't track which daemon a mirai is running on. This will be addressed soon - the next version of mirai will have the ability for `stop_mirai()` to actually cancel/interrupt a task.

**Comment by be-marc** [comment]
Thanks for the quick response. Great, I can wait for the next release.
